doc/exts/pylint_messages.py

```diff
@@ -126,12 +131,21 @@ def _get_all_messages(
         "refactor": defaultdict(list),
         "information": defaultdict(list),
     }
-    for message in linter.msgs_store.messages:
+    checker_message_mapping = chain.from_iterable(
+        ((checker, msg) for msg in checker.messages)
+        for checker in linter.get_checkers()
+    )
+    for checker, message in checker_message_mapping:
         message_data_path = (
             PYLINT_MESSAGES_DATA_PATH / message.symbol[0] / message.symbol
         )
         good_code, bad_code, details, related = _get_message_data(message_data_path)
 
+        checker_module = getmodule(checker)
+
+        if not checker_module or not checker_module.__file__:
```

Comment:
For which checkers does this fail?

Reply:
It should not fall. It was introduced to make mypy happy.

Reply:
Ah, can you use an assert then? That way we will also fail the build process if for some reason this does fail in the future.